dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/lksg/categories/general/subcategories/production/LksgCountryAssociatedSuppliers.kt

```diff
@@ -0,0 +1,14 @@
+package org.dataland.datalandbackend.model.lksg.categories.general.subcategories.production
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+/**
+ * --- API model ---
+ * Fields of the LKSG questionnaire describing the number of suppliers a company has per country
+ */
+data class LksgCountryAssociatedSuppliers(
```

Comment:
Just a proposal:   How about `LksgCountryAssociatedNumberOfSuppliers` ?
I know it is longer, but currently the model name gives you the impression that actual supplier info is included.  But it is just the aggregated number.

Reply:
This will be gone anyways when we switch this to a map.

Reply:
But good proposal